### PR TITLE
Updating BinarySensor to have lastValue as a boolean property

### PR DESCRIPTION
**Pull request by akshayj-MSFT**
(blocking issue for customer)

### CODE REVIEW

File: Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json [hunk 1]
```diff
@@ -5,5 +5,17 @@
     "en": "Binary sensor"
   },
   "extends": "dtmi:digitaltwins:rec_3_3:device:StateSensor;1",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "lastValue",
+      "displayName": {
+        "en": "Last Value"
```
Comment (hammar) [inline Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json:13]: For consistency with the rest of the ontology (and the underlying OWL models) this should be all lowercase, i.e., "last value"

File: Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json [hunk 2]
```diff
@@ -5,5 +5,17 @@
     "en": "Binary sensor"
   },
   "extends": "dtmi:digitaltwins:rec_3_3:device:StateSensor;1",
+  "contents": [
+    {
+      "@type": "Property",
+      "name": "lastValue",
+      "displayName": {
```
Comment (hammar) [inline Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json:12]: For consistency with the rest of the ontology and the underlying OWL models, a description should probably be added, see https://github.com/Azure/opendigitaltwins-building/blob/master/Ontology/Capability/Sensor/QuantitySensor/AreaSensor.json for example.

**Review by hammar — CHANGES_REQUESTED** [review]
Just some minor adjustments to avoid hassle in future regeneration from underlying models.

File: Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json (hunk 1 above)
Comment (akshayj-MSFT) [reply Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json:13]: Got it, will modify

File: Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json (hunk 2 above)
Comment (akshayj-MSFT) [reply Ontology/Capability/Sensor/StateSensor/BinarySensor/BinarySensor.json:12]: Will add description as well

**Pull request merged**